InvenTree/plugin/base/integration/test_mixins.py

```diff
@@ -232,30 +242,32 @@ def test_api_call(self):
         # api_call
         result = self.mixin.get_external_url()
         self.assertTrue(result)
-        self.assertIn('data', result,)
+
+        for key in ['login', 'email', 'name', 'twitter_username']:
+            self.assertIn(key, result)
 
         # api_call without json conversion
         result = self.mixin.get_external_url(False)
         self.assertTrue(result)
         self.assertEqual(result.reason, 'OK')
 
         # api_call with full url
-        result = self.mixin.api_call('https://reqres.in/api/users/2', endpoint_is_url=True)
+        result = self.mixin.api_call('https://api.github.com/orgs/inventree', endpoint_is_url=True)
         self.assertTrue(result)
 
         # api_call with post and data
         result = self.mixin.api_call(
-            'api/users/',
-            data={"name": "morpheus", "job": "leader"},
-            method='POST'
+            'repos/inventree/InvenTree',
+            method='GET'
```

Comment:
@SchrodingersGat this test is not really doing anything valuable anymore as the previous implementation did a POST request to ensure that works too. Could we change that back to ensure all methods work?

Reply:
Can do, I was needing the tests to pass and waiting for your return.

Reply:
Should be resolved in https://github.com/inventree/InvenTree/pull/3328